File: 1470-shuffle-the-array/1470-shuffle-the-array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] ans= new int [2*n];
        int left = 0;
        int right = n;
        for (int i=0;i<n*2;i+=2){
            ans[i] = nums[left];
            ans[i+1] = nums[right];
            left++;
            right++;
        }
        return ans;
    }
}

File: 1470-shuffle-the-array/1470-shuffle-the-array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        ans = [0] *(2*n)
        left = 0
        right = n
        for i in range(0,n*2,2):
            ans[i] = nums[left]
            ans[i+1] = nums[right]
            left +=1
            right+=1
        return ans